modules/local/changeo/changeo_makedb.nf

```diff
@@ -28,19 +28,10 @@ process CHANGEO_MAKEDB {
     path("*_command_log.txt"), emit: logs //process logs
 
     script:
-    if (params.loci == 'ig'){
-        """
-        MakeDb.py igblast -i $igblast -s $reads -r \\
-        ${imgt_base}/${params.species}/vdj/ \\
-        $options.args \\
-        --outname "${meta.id}" > "${meta.id}_command_log.txt"
-        """
-    } else if (params.loci == 'tr') {
-        """
-        MakeDb.py igblast -i $igblast -s $reads -r \\
-        ${imgt_base}/${params.species}/vdj/ \\
-        $options.args \\
-        --outname "${meta.id}" > "${meta.id}_command_log.txt"
-        """
-    }
+    """
+    MakeDb.py igblast -i $igblast -s $reads -r \\
+    ${imgt_base}/${meta.species.toLowerCase()}/vdj/ \\
+    $options.args \\
+    --outname "${meta.id}" > "${meta.id}_command_log.txt"
```

Comment:
We are now using the alsmot the same MakeDb.py igblast call. The only difference now is that I changed the command_log.txt file name from `${meta.id}_command_log.txt` to `${meta.id}_mdb_command_log.txt`. `mdb` stands for MakeDb. If I recall correctly, I had to do this because when the pipeline reached the file size report step, that collects() all command logs, I received a file name collision error.

Reply:
And similar for `changeo_assign_genes.nf`. I generate `${meta.id}_${software}_ag_command_log.txt`. `ag` stands for assign genes.

Reply:
I solve this problem in a different way: when collecting the input channels I place the logs organized within folders: https://github.com/nf-core/bcellmagic/blob/c50843a62c1a2977153bdf9fed9684097d3a561f/modules/local/parse_logs.nf#L22